iChef-WebUi/src/app/components/header/header.component.scss

```diff
@@ -64,3 +64,8 @@
 a:hover {
     cursor: pointer;
 }
+
+::ng-deep.mat-dialog-container {
+    background-color: $primary-background-color !important;
+    border-radius: 25px !important;
+}
```

Comment:
Maybe put this in style.scss since we want this styling to go on every dialog we create